### PR TITLE
featured events 

### PR DESCRIPTION
**Pull request by Mzojobe**

### CODE REVIEW

**Review by Vusumzi-qod — APPROVED** [review]

**Review by codedilla — APPROVED** [review]

**Review by qod-dante — APPROVED** [review]

**Comment by Vusumzi-qod** [comment]
Villa you can merge the PR

**Review by sonwabi — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -23,15 +24,109 @@
...
+                    </div>
+            </div>
+
+            <div class="child">
+            <img src="./images/pexels-sebastian-ervi-1763067.jpg" width="55px" height="30px">
+                <div>
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment </div>      
+                    <div class="share">share</div>
+            </div>
```
Comment (qodvilla) [inline index.html]: Formatting is incorrect.

**Comment by sonwabi** [comment]
what a good code from mzo,wow

File: index.html [hunk 2]
```diff
@@ -23,15 +24,109 @@
...
+                    <div class="share">share</div>
+                </div>
+            </div>
+        </div>
+
+<button type="button">prev</button>
+<button type="button">next</button>
+
+</article>
         </main>
```
Comment (qodvilla) [inline index.html]: Formatting is incorrect.

File: index.html [hunk 3]
```diff
@@ -23,15 +24,109 @@
...
+    <article>
+        <div>
+            <h1>Featured Events</h1>
+                <ul>
+                    <li><a href="/"><h5>Featured</h5></a></li>
+                    <li><a href="/"><h5>Music</h5></a></li>
+                    <li><a href="/"><h5>Theatre</h5></a></li>
+                    <li><a href="/"><h5>Expo & Exhibitions</h5></a></li>
+                    <li><a href="/"><h5>Sport & Movies</h5></a></li>
+                </ul>
```
Comment (qodvilla) [inline index.html]: Incorrect formatting.

File: index.html [hunk 4]
```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>QuiickTik</title>
+     <link href=" https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (qodvilla) [inline index.html]: Formatting incorrect.

**Review by qodvilla — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Vusumzi-qod) [reply index.html]: should he use span instead of div

File: index.html (hunk 2 above)
Comment (Vusumzi-qod) [reply index.html]: or should the button be inside the parent-container

File: index.html (hunk 3 above)
Comment (Vusumzi-qod) [reply index.html]: shouldn't he use anchor instead of ul li and h5

Comment (Vusumzi-qod) [reply index.html]: beside the formatting I think he should use anchor tags only

Comment (qodvilla) [reply index.html]: It's an unordered list, so I think it should be fine.

File: index.html (hunk 2 above)
Comment (qodvilla) [reply index.html]: It's the indentation that I'm worried about.

**Review by qodvilla — APPROVED** [review]

**Pull request merged**